### PR TITLE
Fix typo in docs

### PR DESCRIPTION
**Pull request by halafi**

### CODE REVIEW

File: docs/services.md [hunk 1]
```diff
@@ -658,7 +658,7 @@ const credentials = {
   url: '{s3 url}'
 };
 uploadsClient.createUpload({
-  titleset: `${myUsername}.${myTileset}`,
+  tileset: `${myUsername}.${myTileset}`,
   url: credentials.url,
   name: 'my uploads name',
   private: true
```
Comment (halafi) [inline docs/services.md:664]: the `private` property throws another error:

`message: 'The request body contains the invalid property "private".'`

Comment (andrewharvey) [reply docs/services.md:664]: That's probably #392

File: docs/services.md [hunk 2]
```diff
@@ -658,7 +658,7 @@ const credentials = {
   url: '{s3 url}'
 };
 uploadsClient.createUpload({
-  titleset: `${myUsername}.${myTileset}`,
+  tileset: `${myUsername}.${myTileset}`,
```
Comment (andrewharvey) [inline docs/services.md:661]: Thanks for the PR, the md files are autogenerated from the JS files, so the corresponding line is at https://github.com/mapbox/mapbox-sdk-js/blob/main/services/uploads.js#L105, could you update it there too?

File: docs/services.md (hunk 1 above)
Comment (halafi) [reply docs/services.md:664]: @andrewharvey yep, I think there is a way to satisfy the spec by not setting default value for private on the client, anyway left a comment in #392

File: docs/services.md (hunk 2 above)
Comment (halafi) [reply docs/services.md:661]: done

**Review by andrewharvey — APPROVED** [review]
Thanks

**Pull request merged**